keyboards/clawsome/gamebuddy/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Comment:
If this is pro micro compatible/powered, then ideally this should be caterina by default.

Reply:
@drashna so what do I need to change?

Reply:
```suggestion
BOOTLOADER = caterina
```